### DISCUSSION TITLE
The Tuta client is unpleasant to use

### DISCUSSION THREAD

**Discussion by alstr** [tutao/tutanota#6970] [General]
I want to like Tuta, but it's a chore to use in almost every aspect.

The UI could do with a complete overhaul... offline mode and connection issues need ironing out... inserting images inline is sorely needed... the constant updates need streamlining... I could go on. If I was not a paying customer for several years it would be fine, but it feels like it's barely advanced.

Proton's desktop app by comparison is a joy to use. I'm going to assume it's not as privacy-minded as Tuta, but I don't care as it's good enough. Quantum encryption is all well and good but when 99.9% of your emails are to/from non-Tuta users (the 0.1% being Tuta support) the friction at nearly every point of interaction is not worth it.

**Comment by alstr** [comment]
Just updated the desktop app and oh jeez that new logo is not nice to look at. Putting aside the fact that I don't like the design, what is the purpose of it being visible? It's wasted space.

**Comment by ganthern** [comment]
Hi there :wave:

Two things:
* The logo was visible before, in the same space it is now (and the same size I think)
* we've had inline images for years now, it's in the rich text toolbar that's accessible if you activate HTML mail in the settings.

If you have specific issues with the offline mode, your connection or updating the client, we'd be happy to hear about them, but as it stands, this is not very actionable.

**Reply by alstr** [reply]
Hi,

* Yes, the logo was visible before, but again for no real reason. If I've downloaded the client I known I'm using Tuta. Outlook or Proton don't have a huge logo in the top corner. It's worse though now the design has changed.

* Fair enough about the inline images, didn't see the option buried in the menu. Personally it would be better if the attachment button offered the option to insert inline when choosing an image. Just more intuitive. Similarly the lack of drag and drop support for inline images.

* The offline sync issues I've been trying to get progress on for almost 2 years. See https://github.com/tutao/tutanota/issues/4265. There's not been a tangible update.

**Comment by sami155** [comment]
+1 the serivce is great. But I personally think that the Android app just looks ugly and also unpleasant too use. 

If I would have to say a Deisgn that i like, I would say apps like Thunderbird(K-9), gmail and protonmail do look good.

**Reply by LMSSonos** [reply]
agree on both points, just to add some examples

- Android: Getting into the settings is just hard to hit the small icon, and navigation within settings is not intuitive, should change that navigation similar as the other players do
- Android: Compose an email, it's not really intuitive where to click to enter subject, text etc, think the material components here are kind of outdated, again compared to the others

But in general still like tuta and that they are small

**Comment by MasterInQuestion** [comment]
I use only the Web version.
    And I find the Proton apps less pleasant to use...

    Primarily for resource consumption, speed.
    [ ^ [ https://github.com/tutao/tutanota/issues/8250#issuecomment-2698679190](https://github.com/tutao/tutanota/issues/8250#issuecomment-2698679190) ]
    And the peculiar separated settings page design.
    (separate page from the main app: accessing from the app may interrupt browsing experience)

    See also:
    https://github.com/ProtonMail/WebClients/issues?q=sort:updated+involves:MasterInQuestion
\
\
    On the aesthetics of Tuta UI: I don't find it seriously wrong.
    Though there are indeed some usability issues I would submit later.
    (similar issues also present in Proton UI; about the over-iconization mostly)
